helm/azurefile-csi-driver-app/templates/csi-azurefile-controller.yaml

```diff
@@ -58,6 +73,8 @@ spec:
             - "--leader-election-namespace={{ .Release.Namespace }}"
             - "--timeout=300s"
             - "--extra-create-metadata=true"
+            - "--kube-api-qps=50"
+            - "--kube-api-burst=100"
```

Comment:
Any issues adding those number into `values.yaml`, we might need to change the requests depending on the size of the cluster

Reply:
Sounds good lets do that 👍